watchtower/wtserver/server.go

```diff
@@ -55,6 +55,8 @@ type Config struct {
 
 	// ChainHash identifies the network that the server is watching.
 	ChainHash chainhash.Hash
+
+	NoAckUpdates bool
```

Comment:
Why would we not want to ACK updates?

Reply:
Useful for writing a unit test to simulate a faulty server.